<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>579</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>B2W PDF FORMATER</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ico.ico</normaloff>ico.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextOnly</enum>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="maximumSize">
          <size>
           <width>450</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(125, 125, 125);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLineEdit" name="txtPDF">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>280</width>
            <height>50</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
	border: 2px solid rgb(45,45,45);
	border-radius: 5px;
	padding: 15px;
	background-color: rgb(30, 30, 30);
	color: rgb(100, 100, 100);
	font: 12pt &quot;Arial&quot;;
}</string>
          </property>
          <property name="placeholderText">
           <string>C:\arquivo.pdf</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="txtFolder">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>280</width>
            <height>50</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
	border: 2px solid rgb(45,45,45);
	border-radius: 5px;
	padding: 15px;
	background-color: rgb(30, 30, 30);
	color: rgb(100, 100, 100);
	font: 12pt &quot;Arial&quot;;
}</string>
          </property>
          <property name="placeholderText">
           <string>C:\pasta\salvar</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnFile">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>10</y>
            <width>121</width>
            <height>51</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border: 2px solid rgb(45,45,45);
	border-radius: 5px;
	padding: 15px;
	background-color: rgb(30, 30, 30);
	color: rgb(100, 100, 100);
	font: 8pt &quot;Arial&quot;;
}</string>
          </property>
          <property name="text">
           <string>PDF</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnFolder">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>70</y>
            <width>121</width>
            <height>51</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border: 2px solid rgb(45,45,45);
	border-radius: 5px;
	padding: 15px;
	background-color: rgb(30, 30, 30);
	color: rgb(100, 100, 100);
	font: 8pt &quot;Arial&quot;;
}</string>
          </property>
          <property name="text">
           <string>Onde Salvar?</string>
          </property>
         </widget>
         <widget class="QProgressBar" name="progressBar">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>140</y>
            <width>421</width>
            <height>41</height>
           </rect>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
         <widget class="QPushButton" name="btnActionMagalu">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>200</y>
            <width>421</width>
            <height>61</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border: 2px solid rgb(45,45,45);
	border-radius: 5px;
	padding: 15px;
	background-color: rgb(30, 30, 30);
	color: rgb(100, 100, 100);
	font: 8pt &quot;Arial&quot;;
}</string>
          </property>
          <property name="text">
           <string>Cortar magalu</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnActionB2W">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>280</y>
            <width>421</width>
            <height>61</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border: 2px solid rgb(45,45,45);
	border-radius: 5px;
	padding: 15px;
	background-color: rgb(30, 30, 30);
	color: rgb(100, 100, 100);
	font: 8pt &quot;Arial&quot;;
}</string>
          </property>
          <property name="text">
           <string>Cortar o b2w </string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
